corehq/apps/users/decorators.py

```diff
@@ -74,7 +74,8 @@ def require_permission(permission,
 
 
 require_can_edit_web_users = require_permission('edit_web_users')
-require_can_edit_commcare_users = require_permission(
+require_can_edit_commcare_users = require_permission('edit_commcare_users')
+require_can_edit_and_view_commcare_users = require_permission(
```

Comment:
Confused by this - doesn't edit permission imply view permission?

Reply:
yes, but this implies that there is an option for view only as opposed to just edit